### DISCUSSION TITLE
Test Ether

### DISCUSSION THREAD

**Discussion by ghost** [smartcontractkit/full-blockchain-solidity-course-py#1637] [Q&A]
@jethr0null @ivpavici @freddie71010 @michaelbnewman  Hi sir, Plz help me out !
Etherscan says i had 18.75 rinkeby Ether transaction. But it doesn't show on my wallet. just  showing 18.75 eth received on activity panel.
what i do now ?

**Comment by cromewar** [comment]
Hello @arifulfeni 
The rinkeby network is going to get shutdown so first of all, please change to Goerli.
Now, is possible the token itself, has a different address, so please go to etherscan, copy the token address and add it manually to Metamask.
Also if you want more help, please check #980 .
Cheers.

**Comment by ghost** [comment]
Thank u sir!
it's worked.
